index.d.ts

```diff
@@ -29,6 +29,7 @@ declare namespace JwksRsa {
   interface CertSigningKey {
     kid: string;
     nbf: string;
+    pem: string;
```

Comment:
Not sure `pem` is the right word to use here.  They've been thinking of public key all this time and now we're naming it based on an implementation detail.

Reply:
It is not just that, is part of this library's public API. PEM if the format the "public key" value is returned in. If we happen to change that format to DER, for example, that would break usage as devs would need to make that conversion now.
I think `pem` is widely known in the crypto/keys context. Alternatively, without going with a function, we could name this prop `pubkey` or `pubKey`.

Reply:
then they have `pubKey` and `publicKey`.  I think having a second property with a duplicate value is just plain confusing.  I really think getPublicKey is going to be the best option - as well as being clear that it "gets the public key" for whatever *Key you are using it also allows us to extend it in the future with additional parameters if we ever want to support different formats etc.

Reply:
Yes, that's right. Functions are future proof. I'll have changed this